docs/docs/basil.md

```diff
@@ -128,16 +128,32 @@ Let's start a local ethereum network:
 ganache-cli --deterministic
 ```
 
-This will print 10 accounts. Copy the address of the first one, and then back into the initial terminal, export it as the `OWNER` because it will be useful for us later:
+This will print 10 accounts. Copy the address of the first one and then go back to the terminal window that you used to launch the Ganache client.  Export the address you just copied to an environment variable named `OWNER`.  You will need easy access to that address as you complete this tutorial:
 
 ```sh
 export OWNER=<address>
 ```
 
-Then, to deploy our app:
+Before deploying, make sure your Truffle configuration file contains an entry for a network configuration named `development`.  Your Truffle configuratiom file is named `truffle.js`.  You should see an entry in the `networks` section of that file like the following:
+
+```
+module.exports = {
+  networks: {
+    development: {
+      host: "127.0.0.1",
+      port: 8545,
+      network_id: "*" // Match any network id
+    }
+  }
+};
+```
+
+If you don't, use your favorite editor to add an entry to the `networks` section like the one shown above.  If your `truffle.js` file is empty except for a couple of comment lines, just replace the entire contents of that file with that shown above.  Now Truffle knows where to find the Ganache client.
+
+Next, we deploy our app to Ganache:
```

Comment:
This should actually be covered by the `zos init` command, which generates a suitable truffle config file. We would like to keep these details hidden from the users where possible, so as to avoid cluttering the guides with non-zos content.

Reply:
Removed that section.  `zos init` generates a truffle-config.js file with the port set at 9545 for Ganache in the `networks` section.